nats.go

```diff
@@ -146,6 +146,8 @@ var (
 	ErrInvalidJSAck                 = errors.New("nats: invalid jetstream publish response")
 	ErrMultiStreamUnsupported       = errors.New("nats: multiple streams are not supported")
 	ErrStreamNameRequired           = errors.New("nats: stream name is required")
+	ErrStreamNotFound               = errors.New("stream not found")
```

Comment:
Thanks again for the contribution, looks like it would be ok to include with the `nats: ` prefix to be consistent here:

```suggestion
	ErrStreamNotFound               = errors.New("nats: stream not found")
```